tests/app/templating/summary/test_section.py

```diff
@@ -26,7 +32,7 @@ def test_create_section(self):
         section_schema = {'id': 'section_id', 'title': 'A section title', 'questions': [question_schema]}
 
         # When
-        section = Section('1', section_schema, answers)
+        section = Section(section_schema, answers, "some_link")
 
         # Then
         self.assertEqual(section.id, 'section_id')
```

Comment:
We could add an assertion for the link self.assertEqual(section.link, "some_link")